trade-finance-manager-ui/templates/utilisation-reports/record-corrections/record-correction-log-details.njk

```diff
@@ -4,6 +4,10 @@
 
 {% set backLink = { "href": backLinkHref } %}
 
+{% macro renderSpan(content, dataCy) %}
```

Comment:
I think for macros defined just within a template the naming pattern we have is just to name them as the 'thing' i.e. not have a 'render' prefix